src/bitfield.cpp

```diff
@@ -59,7 +59,7 @@ namespace libtorrent
 	{
 		int ret = 0;
 		const int words = num_words();
-#if TORRENT_HAS_SSE
+#if (TORRENT_HAS_SSE && defined _MSC_VER && _MSC_VER >= 1600)
```

Comment:
good point. this is a problem in a few other places below as well. maybe it would make more sense to make TORRENT_HAS_SSE be set to false for old versions of msvc